backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunDAORdbImpl.java

```diff
@@ -393,51 +396,6 @@ public ExperimentRun insertExperimentRun(ExperimentRun experimentRun, UserInfo u
     return hyrParamMappings;
   }
 
-  @Override
-  public Boolean deleteExperimentRun(String experimentRunId) {
```

Comment:
Where did this go?

Reply:
Not in use so removed it. we have another function called `deleteExperimentRuns` to delete entities.

Reply:
reverted